management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java

```diff
@@ -663,6 +663,75 @@ public Response searchIndexRebuild(
         });
   }
 
+  @POST
+  @Path("/encryption/internode/truststore/reload")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Truststore reloaded successfully",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
+              examples = @ExampleObject(value = "OK")))
+  @Operation(summary = "reload truststore", operationId = "reloadTruststore")
+  public Response reloadTruststore() {
+    return handle(
+        () -> {
+          final String releaseVersion =
+              ResponseTools.getSingleRowStringResponse(
+                  app.dbUnixSocketFile, app.cqlService, CASSANDRA_VERSION_CQL_STRING);
+          if (!releaseVersion.startsWith("4.0.0.68") && !releaseVersion.startsWith("4.0.0.69")) {
+            // rebuilding search index is only possible on DSE
+            return Response.status(Response.Status.BAD_REQUEST)
+                .entity("Reloading the truststore manually is only possible on DSE")
+                .build();
+          }
+
+          app.cqlService.executeCql(
+              app.dbUnixSocketFile, "CALL NodeOps.reloadInternodeEncryptionTruststore()");
+
+          return Response.ok("OK").build();
+        });
+  }
+
+  @GET
+  @Path("/encryption/internode/truststore/issuers")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Got list of truststore issuers",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
+              examples = @ExampleObject(value = "OK")))
+  @Operation(summary = "get internode trustsstore", operationId = "getInternodeTrustStoreIssuers")
+  public Response getInternodeTruststoreIssuers() {
+    return handle(
+        () -> {
+          final String releaseVersion =
+              ResponseTools.getSingleRowStringResponse(
+                  app.dbUnixSocketFile, app.cqlService, CASSANDRA_VERSION_CQL_STRING);
+          if (!releaseVersion.startsWith("4.0.0.68") && !releaseVersion.startsWith("4.0.0.69")) {
+            // rebuilding search index is only possible on DSE
+            return Response.status(Response.Status.BAD_REQUEST)
```

Comment:
issue:
again, the possible 400 returned here isn't documented in the annotation above.